NEWS.md

```diff
@@ -1,3 +1,10 @@
+## 1.0.4 2021-06-11
+
+ * Z39.50 auth errors are handled with more informative message.
```

Comment:
I'm sure I remember seeing a Jira issue requesting this. Shouldn't it be tagged here?